src/overlays/actors/ovl_En_Kz/z_en_kz.c

```diff
@@ -109,26 +109,26 @@ u16 EnKz_GetText(PlayState* play, Actor* thisx) {
     }
 
     if (LINK_IS_ADULT) {
-        return EnKz_GetTextNoMaskAdult(play, this);
+        return EnKz_GetTextIdNoMaskAdult(play, this);
     } else {
-        return EnKz_GetTextNoMaskChild(play, this);
+        return EnKz_GetTextIdNoMaskChild(play, this);
```

Comment:
you can probably remove the "nomask" mention
(this looks (surprise) copypasted from the likes of `EnKo_GetTextId`)